src/pages/tasks/NewTaskDetailsPage.js

```diff
@@ -80,13 +98,17 @@ const NewTaskPage = (props) => {
                         ref={(el) => (inputRef.current = el)}
                         inputID="taskTitle"
                         label={props.translate('newTaskPage.title')}
+                        value={taskTitle}
+                        onChangeText={(text) => setTaskTitle(text)}
                     />
                 </View>
                 <View style={styles.mb5}>
                     <TextInput
                         inputID="taskDescription"
-                        defaultValue=""
+                        defaultValue={props.task.description}
```

Comment:
Same as the previous comment. DefaultValue is redundant here @akinwale

Reply:
@abdulrahuman5196 Done.